new/server/models/user.js

```diff
@@ -3,13 +3,29 @@ const mongoose=require('mongoose');
 const Schema=mongoose.Schema;
 
 const userSchema=new Schema({
+    fname:{
+        type:String,
+        required:true
+    },
+    lname:{
+        type:String,
+        required:true
+    },
+    username:{
+        type:String,
+        required:true
+    },
     email:{
         type:String,
         required:true
     },
     password:{
         type:String,
         required:true
+    },
+    profilepic:{
+        type:String,
```

Comment:
Change the name to image, this sounds weird. Make the changes everywhere where this term is used.

Reply:
corrected